components/common/successfulModal/ShareSocialsSection.vue

```diff
@@ -63,7 +63,7 @@ const { toast } = useToast()
 const { shareOnX, shareOnTelegram, shareOnFarcaster } = useSocialShare()
 
 const handleShareOnX = () => {
-  shareOnX(props.text, props.url)
+  shareOnX(props.text, props.url, '')
```

Comment:
```suggestion
  shareOnX(props.text, props.url, null)
```

Reply:
type mismatch 👀 
<img width="301" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/6c52ae67-8ef1-4f09-bfa2-a3f17214edb5">

Reply:
`via?: string = 'KodaDot'` or `via: string|null = 'KodaDot'`

it's a small thing , I would not add that query param to the url if it's nullish